elixir/apps/web/lib/web/live/gateway_groups/new.ex

```diff
@@ -49,84 +125,4 @@ defmodule Web.GatewayGroups.New do
 
     {:noreply, socket}
   end
-
-  def render(assigns) do
-    ~H"""
-    <.breadcrumbs account={@account}>
-      <.breadcrumb path={~p"/#{@account}/gateway_groups"}>Gateway Instance Groups</.breadcrumb>
-      <.breadcrumb path={~p"/#{@account}/gateway_groups/new"}>Add</.breadcrumb>
-    </.breadcrumbs>
-
-    <.header>
-      <:title :if={is_nil(@group)}>
-        Add a new Gateway Instance Group
-      </:title>
-      <:title :if={not is_nil(@group)}>
-        Deploy your Gateway Instance
```

Comment:
Did you delete this branch of the logic completely? As I remember the idea is that once groups are created we show a message waiting for the first gateway to be connected. The `@group` is assigned during `"submit"` event. And we use similar logic in a few other places.

Reply:
Ah, yeah, I must have just overlooked this and accidentally deleted the branching.  I'll get this put back in.  And yeah, it wasn't caught by tests, so I'll make sure to add a test for it.

Reply:
This should be back in now.